R/get_delay_estimate.R

```diff
@@ -11,6 +11,8 @@
 #' @param reporting_triangle Matrix of the reporting triangle, with rows
 #'   representing the time points of reference and columns representing the
 #'   delays. Can be a reporting matrix or incomplete reporting matrix.
+#'    Can also be a ragged reporting triangle, where multiple columns are
+#'    reported for the same row. (e.g. weekly reporting of daily data).
```

Comment:
Wouldn't weekly reporting of daily data be a long ragged triangle (you have N reference times, and N/7 reporting times)?

Reply:
No I don't think so. This is the short triangle example?

Reply:
Am I losing my mind -- daily data corresponds to the reference times, delays correspond to the report dates. There would be 1/7 of the report dates for each reference time which should make for a long triangle (N x N/7)?

Whereas weekly data with potentially daily updates would be a short triangle (data only assigned to one weekday reference time means N/7 x N matrix)

Reply:
Daily data with weekly reporting requires fitting 7 nowcasts so at the point estimate the delay then matrix will be just weekly reference times and daily report times

Reply:
Weekly reporting with daily data is easy as its just the normal reporting triangles but with meaningful zeros

Reply:
See #110 I am not sure what other use cases you had in mind for this. I think given I don't fully understand the use case anything else might be better addressed in a new issue.